src/main/resources/org/spongepowered/launch/ivysettings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<ivysettings>
+    <settings defaultResolver="sponge" />
+    <resolvers>
+        <chain name="sponge">
+			<ibiblio name="m2" m2compatible="true" />
```

Comment:
Funny, that's what I started with but it seems like the identifiers for some of the artefacts are wrong because it wouldn't pull scala off the forge repo for some reason.